docs/tutorial/02-namespacing.md

```diff
@@ -83,32 +83,32 @@ Using the `namespace` prop is pretty simple, but it doesn't scale well if you ne
 
 There are two approaches to using this component, let's start with the simpler one.
 
-Wrap each widget separately via a `<Provider namespace={namespace} />` element. This is the go-to approach if you are rendering the widgets manually or if the widgets do not share a common React tree, i.e. they are rendered using multiple `ReactDOM.render` calls.
+Wrap each widget separately via a `<NamespaceProvider namespace={namespace} />` element. This is the go-to approach if you are rendering the widgets manually or if the widgets do not share a common React tree, i.e. they are rendered using multiple `ReactDOM.render` calls.
 
 ```js
 import React, { Fragment } from 'react';
-import { Provider } from '@redux-tools/react';
+import { NamespaceProvider } from '@redux-tools/react';
```

Comment:
in `epics-react`, `reducers-react`, and `middleware-react` docs, there is an `_provider.md` partial included at the bottom.

please remove this partial, remove its from each of the packages and instead move its contents to the `namespaces-react` docs

(which, AFAIK, doesn't exist)